lib/datadog/core/telemetry/collector.rb

```diff
@@ -43,11 +43,20 @@ def application
 
         # Forms a hash of standard key value pairs to be sent in the app-started event configuration
         def configurations
+          peer_service_enabled = Datadog.configuration.tracing.contrib.global_default_service_name.enabled
+
           hash = {
             DD_AGENT_HOST: Datadog.configuration.agent.host,
             DD_AGENT_TRANSPORT: agent_transport,
             DD_TRACE_SAMPLE_RATE: format_configuration_value(Datadog.configuration.tracing.sampling.default_rate),
+            DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED: peer_service_enabled
           }
+          peer_service_mapping_str = ''
```

Comment:
Do you need this value to be an empty string, instead of `nil`?

I ask because `nil` will remove it from the Hash, reducing the payload size needed to be flushed.

Reply:
Also, it would make it slightly simpler to write this, given we don't have to initialize `peer_service_mapping_str` anymore if we are ok with `nil`.

Reply:
Other tracers include the empty string since it makes it easier to recognize if the configuration is actually supported